src/main/kotlin/blackjack/domain/result/Winners.kt

```diff
@@ -2,21 +2,35 @@ package blackjack.domain.result
 
 import blackjack.domain.participantion.Dealer
 import blackjack.domain.participantion.Player
+import blackjack.domain.participantion.Price
 
-class Winners(
-    private val players: List<Player>
-) {
-
-    fun exist(name: String): Boolean =
-        players.any { player -> player.name == name }
+class Winners(val players: List<Player>) : Rank(players) {
 
     companion object {
+        private const val BLACK_JACK_TIMES = 0.5
+
         fun from(dealer: Dealer, players: List<Player>): Winners {
-            val winners = players.filter { player ->
-                player.point > dealer.point
-            }
+            val winners = players.filter { player -> player.isWin(dealer) }
 
             return Winners(winners)
         }
     }
+
+    fun blackJackIncome(dealer: Dealer): Price {
+        val blackJackPlayers = getBlackJackPlayers()
+
+        if (dealer.isBlackJack()) {
+            return Price.ZERO
+        }
+
+        val incomeAmount = blackJackPlayers.sumOf { player ->
+            val price = player.price
+            val timesAmount = price.amount.times(BLACK_JACK_TIMES)
+            val blackJackIncome = Price(timesAmount.toInt())
+
+            blackJackIncome.amount
+        }
```

Comment:
```suggestion
   val incomeAmount = blackJackPlayers.sumOf { player ->
            player.price.amount.times(BLACK_JACK_TIMES)
        }
```

이렇게 변경하고 `player.price.amount.times()` 부분은 디미터 법칙을 위반하고 있으니 price에서 계산한 값을 반환해주도록 하면 될 것 같습니다  :)